argilla/src/argilla/datasets/_export/_disk.py

```diff
@@ -117,7 +123,7 @@ def _persist_dataset_model(self, path: Path):
         if path.exists():
             raise FileExistsError(f"Dataset already exists at {path}")
         with open(file=path, mode="w") as f:
-            json.dump(self._model.model_dump(), f)
+            json.dump(self._MODEL.model_dump(), f)
```

Comment:
This won't work. The _model attribute is defined at "Dataset" level. Also, it would be better to user the `.api_model()`
```suggestion
            json.dump(self.api_model().model_dump(), f)
```